events/meeting_handler.py

```
@@ -0,0 +1,57 @@
+import json
+
+class MeetingHandler:
+    rooms = {}
```

Ruff (0.15.18)

[warning] Mutable default value for class attribute

(RUF012)